test/test_tensor_ops.py

```diff
@@ -0,0 +1,411 @@
+import functools
```

Comment:
I wouldn't create a new file. It's a pain and requires registration. 

The OpInfo parts can go into common_methods_invocations.py and the tests can go into test_tensor_creation_ops.py.